api_spec.yaml

```diff
@@ -107,8 +107,98 @@ paths:
         
         '500':
           $ref: '#/components/responses/500'
```

Comment:
missing API for client ID + session handle -> tokenInfo[]

Reply:
changes made